### PR TITLE
Update to readmes, qconsole workspaces

### PR DESCRIPTION
**Pull request by grechaw**
Takes care of README bringing it up to date fro EA-3 eaxmples.  #124 
Also updates QConsole workspaces for demo on Monday.  There will still be version translation in QC next week.

### CODE REVIEW

**Comment by grechaw** [comment]
No code in here, @bsrikan , but if you want to take a look at README for sanity and merge, that would be great

**Comment by grechaw** [comment]
Added another file.

File: README.md [hunk 1]
```diff
@@ -27,7 +31,9 @@ symbolic link from the Mark
 
 or
 
-* A built and installed current nightly MarkLogic package.  If you've used this method, remove or rename the entity-services directory at `src/Modules/MarkLogic/entity-services`
+* A built and installed current nightly MarkLogic package.  If you've used this
+  method, remove or rename the entity-services directory at
+  `src/Modules/MarkLogic/entity-services`
 
 
 For either method of server development, you'll need to symbolically link
```
Comment (bsrikan) [inline README.md]: Should this be removed. We dont have to create a sym link anymore right?

File: entity-services-examples/README.md [hunk 2]
```diff
@@ -1,61 +1,81 @@
...
+ REST extension space.  This code was created from stubs generated by Entity 
+ Services.
+* Transforms go from `src/main/ml-modules/transforms` to REST config.  These
+ transforms wrap calls to the Entity Services modules, but were authored by
+ a developer.
+* Search options are installed from `src/main/ml-modules/options` to REST.
+ This configuration was generated by Entity Services.
+* A module to generate code is installed from
+ `src/main/ml-modules/services` to REST.  This extension is just used by the
+ example ode to invoke the Entity Services API as a demonstration.
```
Comment (bsrikan) [inline entity-services-examples/README.md]: do you mean "example code"? I think it is better to explicitly call out that its authored by a developer here as well, although "demonstration" might suggest that.

File: entity-services-examples/README.md [hunk 3]
```diff
@@ -1,61 +1,81 @@
...
+
+* It loads documents "as-is" from the `data` directory.  When this step is
+ complete, the database will have lots of JSON documents as staged data.
+* It loads some RDF reference data.
+* It invokes an in-place transform batch on the raw documents.  This transform
+ invokes code generated by Entity Services to wrap the JSON documents and store
+ them in their canonical format.
+* Lastly, it actually invokes the code generators and saves the results to the
+ `gen` directory.  Note that this step is conceptually distinct from the
+ previous ones, and would have had to already happen in a typical scenario.
```
Comment (bsrikan) [inline entity-services-examples/README.md]: I didnt understand 2 things here:

- by "previous ones" do you mean bullets in step2?
- what is meant by the last sentence "would have had to already happen in a typical scenario"

File: entity-services-examples/races-qc.xml [hunk 4]
```diff
@@ -114,31 +113,32 @@ let $final-join :=
...
 
 (
-"Raw docs: ", count(cts:search(collection("raw"), cts:true-query())),
-"Non raw: ", count(cts:search(collection(), cts:not-query(cts:collection-query("raw")))),
-"Race envelopes: ", count(//es:instance/Race),
-"Run envelopes: ", count(//es:instance/Run),
+"Raw docs (1354 expected): ", count(cts:search(collection("raw"), cts:true-query())),
+"Non raw (1310 expected): ", count(cts:search(collection(), cts:not-query(cts:collection-query("raw")))),
+"Race envelopes (4 expected): ", count(//es:instance/Race),
+"Run envelopes (1300 expected): ", count(//es:instance/Run),
```
Comment (bsrikan) [inline entity-services-examples/races-qc.xml]: Nice addition of number of docs expected. I was wondering if after run all the docs intended were loaded.

File: README.md (hunk 1 above)
Comment (grechaw) [inline README.md]: Well, anyone who wants to do real-time changes to entity services itself will indeed have to.
Users of the library do not.
This message is for people who want to contribute to the library itself, but you're right, if contributions are to examples then that's not required.

File: entity-services-examples/README.md (hunk 2 above)
Comment (grechaw) [inline entity-services-examples/README.md]: OK

File: entity-services-examples/README.md (hunk 3 above)
Comment (grechaw) [inline entity-services-examples/README.md]: It is unclear, true enough.  I do mean the other bullets, and will rewrite the last bit. My idea was that in a real dev scenario one would generate code first, then be doing all the other stuff.

**Comment by grechaw** [comment]
OK, @bsrikan I've pushed a third commit with edits as you've suggested.

**Comment by bsrikan** [comment]
Looking good. Merging.

**Pull request merged**